main.tf

```diff
@@ -34,6 +34,7 @@ module "orca" {
   db_host_endpoint          = var.db_host_endpoint
   db_user_password          = var.db_user_password
   orca_default_bucket       = var.orca_default_bucket
+  dead_letter_queue_topic_subscription_email = var.dead_letter_queue_topic_subscription_email
```

Comment:
Should probably fix the alignment above this.

Reply:
fixed